docker/hive/start-hive.sh

```diff
@@ -7,7 +7,7 @@ METASTORE_PORT=${METASTORE_PORT:=9083}
 DEFAULT_FS=${DEFAULT_FS:=file:///}
 DB_TYPE=${DB_TYPE:=mysql}
 USE_ATLAS=${USE_ATLAS:=false}
-HIVE_AUTH=${HIVE_AUTH:=NONE}
```

Comment:
Do you think we should change the default?

Reply:
hmmm do u want to pass in terraform instead?

Reply:
it means we need to change all tests and pass it as env var

Reply:
Yes

Reply:
because the e2e tests are coupled to this, im thinking we should leave it as our default

Reply:
ok ill change it